gettsim/parameters/soz_vers_beitr.yaml

```diff
@@ -101,61 +119,59 @@ soz_vers_beitr:
   1998-01-01:
     deviation_from: previous
     ges_krankenv:
-      ag: 0.068
-      an: 0.068
+      durchschnitt: 0.136
   2000-01-01:
     deviation_from: previous
     ges_krankenv:
-      ag: 0.06785
-      an: 0.06785
+      durchschnitt: 0.1357
     ges_rentenv: 0.0965
   2001-01-01:
     deviation_from: previous
     ges_krankenv:
-      ag: 0.0679
-      an: 0.0679
+      durchschnitt: 0.1358
     ges_rentenv: 0.0955
   2002-01-01:
     deviation_from: previous
     ges_krankenv:
-      ag: 0.07
-      an: 0.07
+      durchschnitt: 0.14
   2003-01-01:
     deviation_from: previous
     ges_krankenv:
-      ag: 0.072
-      an: 0.072
+      durchschnitt: 0.144
     ges_rentenv: 0.0975
   2004-01-01:
     deviation_from: previous
     ges_krankenv:
-      ag: 0.071
-      an: 0.071
+      durchschnitt: 0.142
   2005-01-01:
     ges_krankenv:
-      ag: 0.0689
-      an: 0.0689
+      durchschnitt: 0.142
+      allgemein: null
+      ermäßigt: null
+      zusatz: 0
     ges_pflegev:
       standard: 0.0085
       zusatz_kinderlos: 0.0025
     arbeitsl_v: 0.0325
     ges_rentenv: 0.0975
-  2006-01-01:
+  2005-07-01:
     deviation_from: previous
     ges_krankenv:
-      ag: 0.0666
-      an: 0.0756
+      durchschnitt: 0.133
+      zusatz: 0.009
+    note: Allgemeiner Beitragssatz wird nicht mehr gleichmäßig aufgeteilt, Einführung eines Zusatzbeitrags
+    reference: Art. 1 G. v. 15.12.2004, BGBl I S. 3445.
   2007-01-01:
     deviation_from: previous
     ges_krankenv:
-      ag: 0.0695
-      an: 0.0785
+      durchschnitt: 0.139
     arbeitsl_v: 0.021
     ges_rentenv: 0.0995
   2008-01-01:
     ges_krankenv:
-      ag: 0.07
-      an: 0.079
+      allgemein: 0.14
+      zusatz: 0.009
```

Comment:
This Zusatz-parameter is a different parameter than zusatz below (which differs over individuals). The `0.009` rate was apparently called Sonderbeitrag. So I propose to rename it to `sonderbeitrag` and the one below to `mean_zusatzbeitrag`.)

Reply:
[Wikipedia](https://de.wikipedia.org/wiki/Gesetzliche_Krankenversicherung#Beitragssatz_und_Zusatzbeitrag) calls it 'fester Zusatzbeitrag'. I see your point, but this would require even more functions, wouldn't it?

Reply:
N functions does not matter :smile: